core/src/main/java/com/klaytn/caver/tx/manager/TransactionManager.java

```diff
@@ -67,6 +77,86 @@ private TransactionManager(Builder builder) {
         return receipt;
     }
 
+    /**
+     * executes a transaction and receives a receipt for its live result
+     *
+     * @param txType transaction
+     * @return receipt for transaction
```

Comment:
How about adding comment that user will get `null` if there is an error

Reply:
Good point. thanks.